qa/L1_FW_iterators_separate_executor/test.h

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="tensorflow-gpu torchvision mxnet-cu##CUDA_VERSION##"
```

Comment:
this will not work since only specific version of TF is compatible with cuda 10 and by adding tensorflow-gpu to pip_packages you will end up with multiple runs of this test for each version.
You can do the same as I did in #779 instead. I think you could simply add those tests to L2_FW_iterators_perf

Reply:
Done